executor/partition_table_test.go

```diff
@@ -3142,6 +3142,28 @@ partition p2 values less than (10))`)
 	tk.MustQuery("select id from p use index (idx) order by id").Check(testkit.Rows("1", "3", "5", "7"))
 }
 
+func TestAggWithGlobalIndex(t *testing.T) {
+	failpoint.Enable("github.com/pingcap/tidb/planner/core/forceDynamicPrune", `return(true)`)
+	defer failpoint.Disable("github.com/pingcap/tidb/planner/core/forceDynamicPrune")
```

Comment:
I would prefer to not use these failpoints in new test, but rather control the execution with analyze table etc.

Reply:
resolved